data_tools/firebase_client_wrapper.py

```diff
@@ -45,16 +46,18 @@ def get_message_ids(dataset_id):
         ids.append(message.id)
     return ids
 
-# This is a much faster way of doing an entire dataset read than repeated get_message calls
+# This is a much faster way of reading an entire dataset read than repeated get_message calls
```

Comment:
Well spotted opportunity for de-americanisation ;) 
Can you also please delete "read" later into the sentence (... reading and entire dataset **read** than ...)?